yaml/nginx.yaml

```diff
@@ -0,0 +1,96 @@
+#nginx deployment and service configuration
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  #FogFlow cloud components will be deployed inside fogflow namespace
+  namespace: fogflow
+  annotations:
+    kompose.cmd: kompose convert -f docker-compose.yml
+    kompose.version: 1.21.0 (992df58d8)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: nginx
+  name: nginx
+spec:
+  #replicas will make sure that 2 pods running all the time for the deployment
+  replicas: 2 
+  selector:
+    matchLabels:
+      io.kompose.service: nginx
+  minReadySeconds: 10
+  #strategy tells about How do want to update the pods?
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
```

Comment:
maxSurge: 1
      maxUnavailable: 0

Give in percentage 25 each for maxSurge & maxUnavailable